src/DCAttribute.cpp

```diff
@@ -61,15 +61,27 @@ namespace splash
         H5Aclose(attr);
     }
 
-    void DCAttribute::writeAttribute(const char* name, const hid_t type, hid_t parent, const void* src)
+    void DCAttribute::writeAttribute(const char* name, const hid_t type, hid_t parent,
+                                     uint32_t ndims, const Dimensions dims, const void* src)
     throw (DCException)
     {
+        if (name == NULL)
```

Comment:
I guess the same is true for he `ndims` attribute, so I can remove that, too? :)